packages/fx-core/src/component/utils/settingsUtil.ts

```diff
@@ -12,7 +12,10 @@ import {
   TelemetryEvent,
   TelemetryProperty,
 } from "../../common/telemetry";
-import { getProjectSettingPathV3 } from "../../core/middleware/projectSettingsLoader";
+import {
+  getProjectSettingPathV2,
+  getProjectSettingPathV3,
+} from "../../core/middleware/projectSettingsLoader";
```

Comment:
minor: this update of import is not be used.

Reply:
updated, thx!